nvme-print.c

```diff
@@ -4553,6 +4576,8 @@ static void nvme_show_id_ctrl_power(struct nvme_id_ctrl *ctrl)
 		printf(" active_power:");
 		print_ps_power_and_scale(ctrl->psd[i].actp,
 				 nvme_psd_power_scale(ctrl->psd[i].apws));
+		printf("\n            active_power_workload:");
+		print_psd_workload(ctrl->psd[i].apws);
```

Comment:
Don't we need to update `json_nvme_id_ctrl()` too?

Reply:
Yes, it also needs to be updated. Will do thanks.

Reply:
@igaw please check now. I have also fixed idle scale which was not correct for json output.